### PR TITLE
Unexpected metrics

### PR DESCRIPTION
**Pull request by ironoa**
https://github.com/w3f/SecOps/issues/224
https://github.com/w3f/SecOps/issues/225

### CODE REVIEW

**Comment by ironoa** [comment]
missing new unit tests

**Comment by ironoa** [comment]
unit tested

File: config/main.sample.yaml [hunk 1]
```diff
@@ -4,3 +4,6 @@ logLevel: info
 validators:
 - name: test
   address: Gt6HqWBhdu4Sy1u8ASTbS1qf2Ac5gwdegwr8tWN8saMxPt5
+  # expected:
```
Comment (Nexus2k) [inline config/main.sample.yaml:7]: I assume you want to have an actual working sample with the new format?

**Review by Nexus2k — APPROVED** [review]

File: config/main.sample.yaml (hunk 1 above)
Comment (ironoa) [reply config/main.sample.yaml:7]: this is just to say that these are optional values... should it be more specific ?

**Pull request merged**